core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -128,20 +128,20 @@ function getAmperizeHTML(html, post) {
     html = urlService.utils.makeAbsoluteUrls(html, urlService.utils.urlFor('home', true), post.url).html();
 
     if (!amperizeCache[post.id] || moment(new Date(amperizeCache[post.id].updated_at)).diff(new Date(post.updated_at)) < 0) {
-        return new Promise(function (resolve) {
-            amperize.parse(html, function (err, res) {
+        return new Promise((resolve) => {
```

Comment:
Please read this conversation: https://github.com/TryGhost/Ghost/pull/9662#discussion_r192737240
Thanks! :)

I'll also post it on the ES6 migration.

Reply:
@kirrg001 sorry if my message wasn't clear in the other thread. This is definitely a situation where we'd use arrow functions in Ghost-Admin as it's an anonymous function not a named function.

Reply:
Just want to confirm again, is arrow allowed here? Of course syntax is fine, I only mean about coding convention.

Reply:
Yeah agree 👍 It's in this anonymous. Arrow functions can be used. Thanks @kevinansfield